packages/mobile/src/common/constants/validation.ts

```diff
@@ -10,6 +10,11 @@ const MAX_CITY_LENGTH = 30;
 const MAX_DELIVERY_DATA_LENGTH = 256;
 const MAX_SOCIAL_NETWORK_LENGTH = 200;
 
+const MAX_TITLE_POST_LENGTH = 50;
```

Comment:
In this matter, we have some disagreements with the frontend and backend, because each direction did it for itself, therefore, in order to avoid conflicts, I also did it separately for us, no sense make it in the shared if we use it only for us